contributing/release-schedule.md

```diff
@@ -1,84 +1,81 @@
 # Release Schedule
 
-- [Detailed schedule](#detailed-schedule)
-- [Determining if your change is in production](#determining-if-your-change-is-in-production)
-- [Cherry picks](#cherry-picks)
-  - [Cherry pick criteria](#cherry-pick-criteria)
-  - [Process for requesting a cherry pick](#process-for-requesting-a-cherry-pick)
-- [AMP Dev Channel](#amp-dev-channel)
-- [AMP Release Candidate (RC) Channel](#amp-release-candidate-rc-channel)
-- [Release cadence](#release-cadence)
-- [Release freezes](#release-freezes)
+- [Release Channels](#release-channels)
+  - [Weekly](#weekly)
+  - [Long-Term Stable (lts)](#long-term-stable-lts)
+- [Determining if your change is in a release](#determining-if-your-change-is-in-a-release)
+- [Release Cadence](#release-cadence)
+  - [Detailed schedule](#detailed-schedule)
+  - [AMP Dev Channel (Experimental)](#amp-dev-channel-experimental)
+  - [AMP Beta Channel (Beta)](#amp-beta-channel-beta)
+- [Release Freezes](#release-freezes)
 
 A new release of AMP is pushed to all AMP pages every week on Tuesday. **Once a change in AMP is merged into the master branch of the amphtml repository, it will typically take 1-2 weeks for the change be live for all users.**
 
-["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) are used to track the status of current and past releases (from the initial cut to canary testing to production). Announcements about releases are made on the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+## Release Channels
```

Comment:
A table listing the different channels, when they're released and what they contain could be helpful here.

Reply:
@danielrozenberg has plans for something along these lines.